CHANGELOG.md

```diff
@@ -94,8 +94,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   commit to commit. It now either follows the flags `--edit|--no-edit` or it
   gets the mode from `ui.movement.edit`.
 
-* `conflict` revset has been renamed to `conflicts`. With this, all revsets that
-  could potentially return multiple results are named in the plural.
+* `conflict()` and `file()` revsets have been renamed to `conflicts()` and `files()`
+  respectively. With this, all revsets that could potentially return multiple
+  results are named in the plural.
```

Comment:
You'll also need to move this to `[Unreleased]: Deprecations`. Can you send a follow up?

Reply:
Whoops. I forgot to move this after the last release. Sending a follow up now. Thanks for catching!

Reply:
Sent #4532.